### PR TITLE
Adding page for run-mailcap , see, print, edit and compose  with minimal description

### PR DESCRIPTION
**Pull request by flouthoc**
same .md should be copied for `see`

`print`

`compose`

### CODE REVIEW

**Comment by igorshubovych** [comment]
According to our guidelines `see`, `print` and `compose` should have their own pages respectively.

**Comment by flouthoc** [comment]
@igorshubovych this page is for `run-mailcap`.  `see`, `print` and `compose` are subset of `run-mailcap` so mailcap must include description and commands associated to it , so i think you should have a page for `run-mailcap`  and then you can also have detailed description of `see`, `print` and `compose` in detailed manner on different pages. :smile: 

Cheers

**Comment by igorshubovych** [comment]
On Linux when i type `man see`, i will see the proper page.

Currently due the way tldr organized, if you type `tldr see`, you will see nothing.

If you type `tldr run-mailcap` you will see combined page.

So maybe don't use aliases and use action examples instead

```Markdown
- description for see action

`run-mailcup --action=see blah-blah-blah`

- description for compose action

`run-mailcup --action=compose blah-blah-blah`

- description for print action

`run-mailcup --action=print blah-blah-blah`
```

**Comment by rubenvereecken** [comment]
@flouthoc would you mind addressing this? I checked out the actions myself and they all have their own man links but also executable aliases. Given that, it'd be better to have separate tldr pages for them as well. That way `tldr see` will actually help a user.

**Comment by igorshubovych** [comment]
@flouthoc do you need help with addressing this?

**Comment by flouthoc** [comment]
@igorshubovych Sorry i'm not able get what are you guys trying to say ? do you want me to create separate pages for `see` , `compose` and `print` ? if yes , i can do that no problem :smile: 

Cheers

**Comment by igorshubovych** [comment]
There are 2 options:

- Have run-mailcup.md page with commands like `run-mailcup --action=see blah-blah-blah`
- Create see.md, print.md, compose.md pages.

Actually you can have both.

**Comment by ostera** [comment]
@igorshubovych: Not a fan of the `--actions` param. We don't do `git --actions=add`, we do `git-add` or `git add`. We should refer to the sub-commands by their name.

@flouthoc: man! thanks for the contribution :) the 3 pages would be the best, and since they are all closely related I think it shouldn't be a problem if you reused this PR, looking forward to merge it soon!

**Comment by flouthoc** [comment]
@ostera @igorshubovych @rubenvereecken  okay few changes made , check them :smile: 
Cheers !

**Comment by igorshubovych** [comment]
Plz check linter's output

**Comment by flouthoc** [comment]
@ostera  actually i think @igorshubovych is right as `see` `compose` , `print` etc are just aliases to run-mailcap's original actions like `see {{filename}}` is equivalent to `run-mailcap --action=view {{filename}}` so run-mailcap's  man page should contain detailed explanation about using --action flag and minimal description about `see` `compose` etc and man pages of `see` , `compose` etc should contain abstract view about directly using alias , so i've made few changes accordingly but travis check are doing few problem

**Comment by flouthoc** [comment]
@igorshubovych  are you talking to travis checks ?

**Comment by igorshubovych** [comment]
Yes.

**Comment by igorshubovych** [comment]
Oops. 33 similarly named commits.

Do you need any help with it?

**Comment by flouthoc** [comment]
@igorshubovych  lol : :sweat_smile: , grammatical errors hard to spot , are we done now it says all checks have passed ?

**Comment by flouthoc** [comment]
@igorshubovych  can you tell me , is it all fine now ? do i still have lexical errors in see.md print.md , compose.md or they all are clear now  ?  :pray:

File: pages/linux/run-mailcap.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`run-mailcap --action=ACTION [--option[=value]]`
+
+- In simple language:
+
+`run-mailcap --action=ACTION {{filename}}`
+
+- Turn on extra information:
+
+`run-mailcap --debug --action=ACTION {{options}}`
```
Comment (igorshubovych) [inline pages/linux/run-mailcap.md]: This example is different from the previous. You added `--debug` option. But why this one ends with `{{options}}` and the previous ends with `{{filename}}`?

File: pages/linux/run-mailcap.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Ignore any "copiousoutput" directive and forward output to STD‐OUT:
+
+`--nopager`
+
+- Display the found command without actually executing it:
+
+`--norun`
```
Comment (igorshubovych) [inline pages/linux/run-mailcap.md]: Why do you have option without command. It should be the full working example.

File: pages/linux/run-mailcap.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`run-mailcap --action=ACTION {{filename}}`
+
+- Turn on extra information:
+
+`run-mailcap --debug --action=ACTION {{options}}`
+
+- Ignore any "copiousoutput" directive and forward output to STD‐OUT:
+
+`--nopager`
```
Comment (igorshubovych) [inline pages/linux/run-mailcap.md]: Why do you have option without command. It should be the full working example.

File: pages/linux/compose.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# compose
+
+> An alias to a run-mailcap's action compose.
```
Comment (igorshubovych) [inline pages/linux/compose.md]: IMO it is better here to mention `run-mailcup` with backticks in description.
```md
> An alias to a `run-mailcap`'s action compose.
```

File: pages/linux/compose.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+> Originally run-mailcap is used to  mime-type/file.
+
+- Compose action can be used to compose any existing file or new  on default mailcap edit tool:
+
+`compose {{filename}}`
+
+- With run-mailcap:
+
+`run-mailcap --action=compose {{filename}}`
```
Comment (igorshubovych) [inline pages/linux/compose.md]: @rubenvereecken, @ostera 
Do you think about having `run-mailcup` examples here?

IMO this is slightly breaking man-style of our pages.

File: pages/linux/run-mailcap.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- In simple language:
+
+`run-mailcap --action=ACTION {{filename}}`
+
+- Turn on extra information:
+
+`run-mailcap --debug --action=ACTION {{options}}`
+
+- Ignore any "copiousoutput" directive and forward output to STD‐OUT:
```
Comment (igorshubovych) [inline pages/linux/run-mailcap.md]: What is "copiousoutput"?

File: pages/linux/run-mailcap.md (hunk 1 above)
Comment (flouthoc) [inline pages/linux/run-mailcap.md]: yes i changed run-mailcap.md a bit cause i thought , it would be better to talk about run-mailcap in run-mailcap's man page rather than talking about aliases , {{options}}is a typo

File: pages/linux/run-mailcap.md (hunk 6 above)
Comment (flouthoc) [inline pages/linux/run-mailcap.md]: in layman language - Ignoring "copiousoutput" - means to give only small summary of command and ignore abundant content in output

**Comment by flouthoc** [comment]
@igorshubovych alright changes made, can you review again ?

**Comment by igorshubovych** [comment]
@flouthoc

Thanks a lot!

I decided to merge it on this stage, we can continue working on this commands in other PRs.
I merged it with patch, obviously I could not afford to merge 39 similarly-annotated commits. Your name is tagged to commit annotation.

Here is the commit - 027402e8e7c488cec64da1dbb317c02bdad79442.

**Comment by igorshubovych** [comment]
For future, please always use feature branches.

always

Here is the [short instruction how to do it](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

**Comment by rubenvereecken** [comment]
Thanks @igorshubovych for handling the merge yourself.
